recipes/_configure_authentication.rb

```diff
@@ -108,7 +108,11 @@
     if hash['bindDNpassword']
       password = CernerSplunk::DataBag.load hash['bindDNpassword'], default: default_coords, type: :vault
       fail 'Password must be a String' unless password.is_a?(String)
-      hash['bindDNpassword'] = password
+      ruby_block 'set bindDNpassword' do
```

Comment:
Same comment as https://github.com/cerner/cerner_splunk/pull/121/files#r65924306 

Refer to the strategy name in this resource name if we want to do things this way.

Alternatively... should we have an enhancement to the app resource to loop over all of the k-v pairs and evaluate  value lambdas? 

so this becomes 

    hash['bindDNpassword'] = -> { CernerSplunk.splunk_encrypt_password password, node.run_state['cerner_splunk']['splunk.secret'] }

The question becomes if where we would evaluate the lambda, do we have access to the node run state? Could also have a method that produces such lambdas, currying in the password and if the xor needs to be done), but *shrug*